packages/headless/src/features/result-templates/result-templates-helpers.test.ts

```diff
@@ -114,6 +128,13 @@ describe('result template helpers', () => {
       expect(match(result)).toBe(true);
     });
 
+    it(`when sending a field that does not contain any of the values (as part of the root result) to match
+    should return true`, () => {
+      const match = fieldMustNotMatch('excerpt', ['test']);
+      const result = buildMockResult({excerpt: 'no!'});
+      expect(match(result)).toBe(true);
+    });
```

Comment:
Are there any tests for when the result property value is `null`?

Reply:
I don't think the API itself returns null, rather an object with inexistant keys

Reply:
@acote-coveo For the API, can the result return `null` or `NaN` or `undefined` or it does just omit the non existant values?